samples/http_example/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "nodejs-docs-samples-iot-http-example",
```

Comment:
Generally we try to roll with a single `samples/package.json` file that captures all of the stuff you need to run any of the samples.  Is it ok to combine these in to one?

Reply:
I have a slightly different opinion on this one, agreed for most cases we should just roll all the dependencies up into a top level package.json, e.g., if you're introducing a new dependency like `uuid`.

If the sample is a fully self contained application, the idea being that a customer use it as a starting point for their own application, I'm comfortable with an additional `package.json`.

This should be for extenuating circumstances, i.e., such as a fully functional example express server. I'm not sure if this falls under that or not?

Reply:
Each sample is atomic except for testing in which there are inter-dependencies that are baked into the test bootstraps. I think this is resolved.